src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -45,19 +47,24 @@
  * @author Peter Güttinger
  */
 @Name("Last Spawned Entity")
-@Description("Holds the entity that was spawned most recently with the spawn effect, drop with the <a href='../effects/#EffDrop'>drop effect</a> or shot with the <a href='../effects/#EffShoot'>shoot effect</a>. " +
+@Description("Holds the entity that was spawned most recently with the spawn effect (section), dropped with the <a href='../effects/#EffDrop'>drop effect</a>, shot with the <a href='../effects/#EffShoot'>shoot effect</a> or created with the <a href='../effects/#EffLightning'>lightning effect</a>. " +
```

Comment:
It's not explicitly for the spawn effect section is it?

Reply:
No, that's why section is in parentheses

Reply:
Oh I see, I misunderstood this originally